tools/api-markdown-documenter/src/test/snapshots/html/simple-suite-test/sparse-config/test-suite-a/testfunctionreturningintersectiontype-function.html

```diff
@@ -1,47 +1,24 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
   <head>
-    <meta charset="utf-8" />
+    <meta charset="utf-8">
   </head>
   <body>
     <section>
-      <h2>
-        testFunctionReturningIntersectionType
-      </h2>
+      <h2>testFunctionReturningIntersectionType</h2>
       <section>
-        <p>
-          Test function that returns an inline type
-        </p>
+        <p>Test function that returns an inline type</p>
       </section>
       <section>
-        <p>
-          <span><b>WARNING: This API is deprecated and will be removed in a future release.</b></span>
-          <br>
-          <span>
-          <p>
-            <i>This is a test deprecation notice. Here is a</i> <a href='docs/test-suite-a/testfunctionreturninguniontype-function'><i>link</i></a> <i>to something else!</i>
-          </p>
-          </span>
-        </p>
+        <p><span><b>WARNING: This API is deprecated and will be removed in a future release.</b></span><br><span><i><p>This is a test deprecation notice. Here is a <a href="docs/test-suite-a/testfunctionreturninguniontype-function">link</a> to something else!</p></i></span></p>
       </section>
       <section>
-        <h3 id="testfunctionreturningintersectiontype-signature">
-          Signature
-        </h3>
-        <code>
-          export declare function testFunctionReturningIntersectionType(): TestEmptyInterface &amp; TestInterfaceWithTypeParameter&lt;number&gt;;
-        </code>
+        <h3 id="testfunctionreturningintersectiontype-signature">Signature</h3><code>export declare function testFunctionReturningIntersectionType(): TestEmptyInterface &#x26; TestInterfaceWithTypeParameter&#x3C;number>;</code>
       </section>
       <section>
-        <h3 id="testfunctionreturningintersectiontype-returns">
-          Returns
-        </h3>
-        <p>
-          an intersection type
-        </p>
-        <p>
-          <span><b>Return type:</b> </span><span><a href='docs/test-suite-a/testemptyinterface-interface'>TestEmptyInterface</a> &amp; <a href='docs/test-suite-a/testinterfacewithtypeparameter-interface'>TestInterfaceWithTypeParameter</a>&lt;number&gt;</span>
-        </p>
+        <h3 id="testfunctionreturningintersectiontype-returns">Returns</h3>
+        <p>an intersection type</p>
+        <p><span><b>Return type: </b></span><span><a href="docs/test-suite-a/testemptyinterface-interface">TestEmptyInterface</a> &#x26; <a href="docs/test-suite-a/testinterfacewithtypeparameter-interface">TestInterfaceWithTypeParameter</a>&#x3C;number></span></p>
```

Comment:
Mangled `&lt;` and (incorrectly, I think) transformed `&gt;` into `>`.

Reply:
This escaping appears to be valid.

![image](https://github.com/user-attachments/assets/0f4a37f7-c053-4fef-acbf-5489c862e399)